stl/inc/xloctime

```diff
@@ -760,6 +768,17 @@ protected:
     }
 
 private:
+    static constexpr bool __CLRCALL_OR_CDECL _Is_valid_fmt_specifier(char _Specifier) {
+        constexpr char _Valid_specifiers[] = "aAbBcCdDeFgGhHIjmMnprRStTuUVwWxXyYzZ";
+        for (const char _Valid_specifier : _Valid_specifiers) {
+            if (_Valid_specifier == _Specifier) {
+                return true;
+            }
+        }

```

Comment:
I was thinking about binary search here, but I guess this would require movement to a .cpp file which as of now does not exist for `xloctime`. I'm accepting any guidelines or suggestions.